src/locale/locales/pt-BR/messages.po

```diff
@@ -766,15 +766,15 @@ msgstr "Tem certeza de que deseja excluir este kit inicial?"
 
 #: src/components/dms/LeaveConvoPrompt.tsx:48
 msgid "Are you sure you want to leave this conversation? Your messages will be deleted for you, but not for the other participant."
-msgstr ""
+msgstr "Tem certeza de que deseja sair desta conversa? Suas mensagens serão excluídas para você, mas não para os outros participantes."
 
 #: src/view/com/feeds/FeedSourceCard.tsx:313
 msgid "Are you sure you want to remove {0} from your feeds?"
 msgstr "Tem certeza que deseja remover {0} dos seus feeds?"
 
 #: src/components/FeedCard.tsx:310
 msgid "Are you sure you want to remove this from your feeds?"
-msgstr ""
+msgstr "Tem certeza que deseja remover dos seus feeds?"
```

Comment:
Maybe `"Tem certeza que deseja remover isto de seus feeds?"` ? This seems to be used while confirming a removal action; the `isto` makes it more explicit

Reply:
ok